Uppsala/UppEt18.xml

```diff
@@ -0,0 +1,190 @@
+                                is an erased stanza after the last one, leading him to suggest that the text here is incomplete.
+                            </note>
+                            
+                            <incipit xml:lang="gez">
+                                ሰላም፡ ለዝክረ፡ ስምከ፡ ምስለ፡ ስመ፡ ልዑል፡ ዘተሳተፈ፡ 
+                                ወልደ፡ ያሬድ፡ ሄኖክ፡ በከመ፡ ጸሐፈ፡ 
+                                ረድአ፡ ምንዱባን፡ ሚካኤል፡ በከመ፡ ታለምድ፡ ዘልፈ፡ 
+                                ለረዲኦትየ፡ ነዓ፡ ሰፊሐከ፡ ክንፈ፡ 
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                ሰላም፡ ለሕሊናከ፡ ዘኢይሔሊ፡ ኀጕለ፡ 
+                                እንበለ፡ ዳእሙ፡ ይስእል፡ ምሕረተ፡ ወሣህለ፡ 
+                                ሚካኤል፡ ንሣእ፡ ወልታከ፡ ፅዱለ፡ 
+                                ወምላ<choice><sic>ዕ</sic><corr resp="PRS6337Lofgren">ኅ</corr></choice>፡ ዲበ፡ መስቴማ፡ ሰይፈከ፡ ስሁለ፡ 
+                                ለዘተፃብዐከ፡ ቅድመ፡ ኢትርሳእ፡ በቀለ፡ 
+                            </explicit>
+                        </msItem>
+                            
+                            <msItem xml:id="ms_i2">
+                                <locus from="18" to="35"/>
+                                <title type="complete" ref="LIT2839RepCh123"></title>
+                                <textLang mainLang="gez"/>
+                                <note><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">85</citedRange></bibl>
+                                    notes that this attestation of the text contains 55 stanzas.
+                                </note>
+                                <incipit xml:lang="gez">
+                                    ሰላም፡ ለዝክረ፡ ስምከ፡ ስመ፡ መሐላ፡ ዘኢይኄሱ፡ 
+                                    ዘአንበረ፡ ቅድመ፡ እግዚአብሔር፡ በአትሮንሱ፡ 
+                                    ኢየሱስ፡ ክርስቶስ፡ ለዳዊት፡ ባሕርየ፡ ከርሡ፡ 
+                                    አክሊለ፡ ስምከ፡ እንዘ፡ ይትቄጸል፡ በርእሱ፡ 
+                                    አሕጉረ፡ ፀር፡ ወረሰ፡ ኢያሱ፡ 
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    ሰላም፡ ሰላም፡ ለኩሉ፡ መልክዕከ፡ 
+                                    ወበከመ፡ መልክዑ፡ ለአቡከ፡ 
+                                    ኢየሱስ፡ ክርስቶስ፡ ሕማማተ፡ መስቀልከ፡ ዘፆርከ፡ 
+                                    ህብዐኒ፡ እምገጸ፡ ሞት፡ በእንተ፡ ማርያም፡ እምከ፡ 
+                                    አምላኪየ፡ አምላኪየ፡ ኢይፅናዕ፡ ልብከ፡ 
+                                </explicit>
+                            </msItem>
+                      
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">39</measure>
```

Comment:
quantity

Reply:
I added this in all Uppsala manuscripts (which should prevent me from doing the same mistake again :) )